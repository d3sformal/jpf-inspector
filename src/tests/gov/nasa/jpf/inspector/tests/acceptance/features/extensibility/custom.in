print1
print2