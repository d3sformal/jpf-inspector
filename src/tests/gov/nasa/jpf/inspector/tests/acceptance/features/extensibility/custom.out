The custom command has been executed.
The custom command has been executed.